Dockerfile.visual-regression

```diff
@@ -0,0 +1,47 @@
+FROM node:16.15.1
+
+ARG BACKSTOPJS_VERSION
+
+ENV \
+	BACKSTOPJS_VERSION=$BACKSTOPJS_VERSION
+
+# Base packages
+RUN apt-get update && \
+	apt-get install -y git sudo software-properties-common
+
+RUN /bin/bash -c 'set -ex && \
+    ARCH=`uname -m` && \
+    if [ "$ARCH" == "x86_64" ]; then \
+       sudo npm install -g --unsafe-perm=true --allow-root backstopjs@${BACKSTOPJS_VERSION}; \
+    else \
+       sudo PUPPETEER_SKIP_DOWNLOAD=true PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true npm install -g --unsafe-perm=true --allow-root backstopjs@${BACKSTOPJS_VERSION} && \
+       apt-get -qqy update && apt-get -qqy --no-install-recommends install chromium && \
+       sudo test -f /usr/bin/chromium && sudo ln -s /usr/bin/chromium /usr/bin/chromium-browser && sudo ln -s /usr/bin/chromium /usr/bin/chrome; \
+    fi'
```

Comment:
This seems to work great on my M1! 

I am, however, a little nervous about merging it this week as I feel like it should be more thoroughly tested from other team members who are running on x86_64 rather than someone who is about to leave for 2 months 😊 . I also wonder whether this might affect the consistency of the screenshots across team members? The nice thing about the current setup is I have yet to see screenshots produced on my machine look any different from those produced on https://pixel.wmcloud.org/reports/desktop/index.html). We are all comparing apples-to-apples. Would this affect that?

For example, could people with M1 macs be download different versions of chromium than those with x86_64 and would that cause different screenshots/results/confusion? Relatedly, how much randomness does putting this logic in a Dockerfile (rather than downloading the backstop Docker image) introduce and would that cause any inconsistency in the screenshots (e.g. team members who build this Docker image at different dates)? I'm mostly thinking out loud here but those are my thoughts